src/app/services/task.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+import { Task } from 'src/app/models/Task.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TaskService {
+  taskList: Task[] = [];
```

Comment:
This service essentially only stores the tasks, whereas adding task should also be its responsibility.

Reply:
addTaskToTaskList(taskList: Task) {
    this.taskList.unshift(taskList);
  }
}
added one function to taskService to add task